openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -179,6 +181,16 @@ def update_account_settings(requesting_user, update, username=None):
             }
             del update[read_only_field]
 
+    additional_fields = {"sch_org","phone", "organization_type"}.intersection(set(update.keys()))
+    if additional_fields:
+        try:
+            instance = AdditionalRegistrationFields.objects.filter(user=existing_user).first()
+            for field in list(additional_fields):
+                setattr(instance, field, update[field])
+            instance.save()
+        except ObjectDoesNotExist:
+            log.info('Addition fields record does not exist')
```

Comment:
According to my understanding, the exception will never be called and if in any scenario the object doesn't exist then an `AttributeError` will be generated while `setattr` method call.
Kindly consider the below code if it seems appropriate to you. You can also add `AttributeError` exception in below code if you feel there will be any scenario where instance may not have the desired field in it.
```suggestion
        instance = AdditionalRegistrationFields.objects.filter(user=existing_user).first()
        if instance:
            for field in list(additional_fields):
                setattr(instance, field, update[field])
            instance.save()
        else:
            log.info('Addition fields record does not exist')
```